src/pip/_internal/req/__init__.py

```diff
@@ -57,11 +57,8 @@ def install_given_reqs(
 
     with indent_log():
         for requirement in to_install:
-            if requirement.conflicts_with:
-                logger.info(
-                    'Found existing installation: %s',
-                    requirement.conflicts_with,
-                )
+            if requirement.should_reinstall:
+                logger.info('Attempting uninstall: %s', requirement.name)
```

Comment:
We're losing information about the version that's installed.

Reply:
It should be provided by the upcoming log `logger.info('Found existing installation: %s', dist)`

Reply:
Ah fair. So much for reviewing PRs on my phone. :)